dotCMS/src/main/java/com/dotmarketing/servlets/BinaryExporterServlet.java

```diff
@@ -162,10 +175,30 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp) throws Servl
 			}
 		}
 		params = sortByKey(params);
-
+		boolean byInode = params.containsKey("byInode");
+		// Default to a no-shortyId value
+		ShortyId shorty = shortyIdApi.noShorty(uuid);

```

Comment:
this too

Reply:
No it cannot. That variable can be changed in lines 185 and 196.